lib/types/gpt.nix

```diff
@@ -100,6 +100,8 @@ in
       default = ''
         ${lib.concatStrings (map (partition: ''
           sgdisk \
+            --set-alignment=2048 \
```

Comment:
Should we make this configurable? I know there are some arm boards that sometimes have weird requirements for embedded uboot loader.

Reply:
The `--set-alignment` kicks in only when you specify relative size (100%) and the default behavior gives you maximum partition size. The only downside with this change is that you might possibly waste for example 0.5 MiB at the end. You can still specify `start = "1234";` for ARM boards and create unaligned partition if that's what you need.

Reply:
Sounds good.